pkg/server/router/operation.go

```diff
@@ -47,15 +48,48 @@ type GetOperationsRequest struct {
 	Parent string `json:"parent"`
 
 	// A standard filter expression conforming to https://google.aip.dev/160.
-	// For example: 'status = done'.
+	// For example: `done = true`.
 	Filter string `json:"filter"`
 }
 
-func (r GetOperationsRequest) ToServiceRequest() operation.GetOperationsRequest {
-	return operation.GetOperationsRequest{
-		Parent: r.Parent,
-		Filter: r.Filter,
+func (r GetOperationsRequest) GetFilter() string {
+	return r.Filter
+}
+
+const (
```

Comment:
nit: could define a type for these e.g. `type OperationsFilter string` however the type conversion may be a pain

Reply:
I don't expect these to be used elsewhere so leaving as is makes sense to me for now.